examples/android/CHIPTool/app/src/main/java/com/google/chip/chiptool/clusterclient/SensorClientFragment.kt

```diff
@@ -181,7 +181,7 @@ class SensorClientFragment : Fragment() {
       return
 
     try {
-      ChipClient.getDeviceController(requireContext()).shutdownSubscriptions(subscribedDevicePtr)
+      ChipClient.getDeviceController(requireContext()).shutdownSubscriptions(true)
```

Comment:
Maybe add an inline comment here? 

```
shutdownSubscriptions(/* schedule= */ true)
```

Reply:
rollback the version without schedule and update the comments. thanks